build/salva2d-f64/Cargo.toml

```diff
@@ -0,0 +1,57 @@
+[package]
+name    = "salva2d-f64"
+version = "0.9.0"
+authors = [ "Sébastien Crozet <developer@crozet.re>" ]
+description = "2-dimensional particle-based fluid dynamics in Rust."
+documentation = "https://salva.rs/docs"
+homepage = "https://salva.rs"
+repository = "https://github.com/dimforge/salva"
+readme = "README.md"
+categories = [ "science", "game-development", "mathematics", "simulation", "wasm"]
+keywords = [ "physics", "dynamics", "particles", "fluids", "SPH" ]
+license = "Apache-2.0"
+edition = "2021"
+
+[badges]
+maintenance = { status = "actively-developed" }
+
+[features]
+default = [ "dim2", "f64" ]
+dim2    = [ ]
+f64 = [ ]
+parallel = [ "rayon" ]
+sampling = [ "rapier" ]
+rapier = [ "parry", "rapier2d" ]
+rapier-testbed = [ "rapier", "rapier_testbed2d", "graphics" ]
+rapier-harness = [ "rapier-testbed" ]
+parry = [ "parry2d" ]
+wasm-bindgen = [ "rapier2d/wasm-bindgen" ]
+graphics = [ "bevy", "bevy_egui" ]
+
+[lib]
+name = "salva2d_f64"
+path = "../../src/lib.rs"
+required-features = [ "dim2", "f64" ]
+
+[dependencies]
+approx = "0.5"
+num-traits = "0.2"
+fnv = "1.0"
+itertools = "0.13"
+generational-arena = "0.2"
+instant = { version = "0.1", features = [ "now" ] }
+rayon = { version = "1.8", optional = true }
+
+nalgebra = "0.33"
+parry2d = { version = "0.16", optional = true }
+rapier2d = { version = "0.21", optional = true }
+rapier_testbed2d = { version = "0.21", optional = true }
```

Comment:
From what I understand, these are your problematic dependencies ? (we'd need to target f64 variants, and testbed doesn´t support those.

Would commenting out the testbeds from the salva f64 version help with making progress without being blocked on a new crate?